src/sqlparser/tests/sqlparser_common.rs

```diff
@@ -3129,6 +3129,69 @@ fn parse_create_materialized_view_emit_immediately() {
     }
 }
 
+#[test]
+fn parse_create_table_on_conflict() {
```

Comment:
please also add pareser test and you can refer https://github.com/risingwavelabs/risingwave/blob/75735e2b15c82429d69529683d56136fc80de1bd/src/sqlparser/README.md